modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/nw-nodes-peer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-egress_{context}"]
+= Egress traffic control for AdminNetworkPolicy and BaselineAdminNetworkPolicy
+
+As a cluster administrator you can use `AdminNetworkPolicy` (ANP) resource in conjunction with `BaselineAdminNetworkPolicy` (BANP) resource to secure and limit egress traffic. Using these namespace scoped policies together you can do the following:
+
+* Create egress traffic control for cluster nodes.
+
+* Apply egress traffic control towards Kubernetes API servers.
+
+* Implement egress traffic control towards external destinations.
+
+//For example, as a cluster administrator, you can enforce security for northbound traffic by restricting traffic from specific cluster workloads to all or specific destinations outside the cluster. Or, you can setup policies that grant access to your cluster-external DNS server even if namespace admins create `NetworkPolicy` resources that block cluster-external egress.
+
+[id="egress-traffic-control-towards-cluster-nodes_{context}"]
+== Using egress traffic control for ANP and BANP
+
+Using the `matchLabels` and `matchExpressions` fields, cluster administrators can select nodes in a cluster based on `nodes` and `networks` labels and apply egress traffic policy to these nodes. The following are benefits to using label selectors when blocking access from cluster workloads to specific ports on cluster nodes:
+
+* You do not have to block access to those ports on external hosts.
+
+* You do not have to manually list the IP address of every node.
+
+* You do not have to change the policy when new nodes are added to the cluster.
+
+The following example applies `matchExpression` key value pairs to deny egress traffic from tenants who do not have a `restricted`, `confidential`, or `internal` level of security. In the example, the `Deny` action is applied using the `nodes.matchLabels` field so that security clearance is needed to access control-plane nodes at `443` and `6443` ports in the cluster.
+
+.Example of ANP `Deny` egress to control-plane
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: egress-security-deny
+spec:
+  priority: 55
+  subject:
+    namespaces:
+      matchExpressions:
+        - {key: security, operator: notIn, values: [restricted, confidential, internal]} <1>
+  egress:
+  - name: "deny-all-egress-to-kapi-server"
+    action: "Deny"
+    to:
+    - nodes: <2>
+        matchLabels:
+          node-role.kubernetes.io/control-plane:
+    ports:
+      - portNumber:
+          protocol: TCP
+          port: 443
+      - portNumber:
+          protocol: TCP
+          port: 6443
+----
+<1> Specifies key/value pairs for `matchExpressions` field.
```

Comment:
You may want to use matchExpression in the example as you are describing it.
<pre>
- nodes:
    matchExpressions:
     - key: node-role.kubernetes.io/control-plane
       operator: Exists
</pre>